packages/plugin-link/src/components/LinkComponent.ts

```diff
@@ -0,0 +1,49 @@
+import { OwlComponent } from '../../../plugin-owl/src/ui/OwlComponent';
+import { InlineNode } from '../../../plugin-inline/src/InlineNode';
+import { LinkFormat } from '../LinkFormat';
+import { LinkParams } from '../Link';
+import { Layout } from '../../../plugin-layout/src/Layout';
+import { useState } from '@odoo/owl';
+
+export class LinkComponent<T = {}> extends OwlComponent<T> {
+    static components = {};
+    static template = 'link';
+    state = useState({
+        url: this._currentUrl(),
+        label: this._currentLabel(),
```

Comment:
It's weird. Those are functions but it have no verbs. Shouldn't they be named something like `_getCurrentUrl` instead ?